src/types/Session.ts

```diff
@@ -1,4 +1,12 @@
+import { Context } from "./Warrant";
+
 export interface SessionParams {
     userId: string;
     tenantId?: string;
+    ttl?: number;
+    context?: Context;
```

Comment:
Technically aren't the only fields for session params `userId` and `ttl`? (maybe context?)

Reply:
Yeah, now that you mention it, I think `tenantId` belongs on `SelfServiceSessionParams`